src/camera.cpp

```diff
@@ -454,7 +454,12 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 	if (player->getPlayerControl().zoom && player->getZoomFOV() > 0.001f) {
 		fov_degrees = player->getZoomFOV();
 	} else {
-		fov_degrees = m_cache_fov;
+		// Default to m_cache_fov if server-sent FOV == 0
+		f32 fov_received = player->getFOV();
+		if (!fov_received)
```

Comment:
Please compare against `== 0` since it's a float. It would be fine for integers but here it might show possible imprecision issues.